docs/en/query_language/select.md

```diff
@@ -564,29 +564,37 @@ ClickHouse doesn't directly support syntax with commas, so we don't recommend us
 
 Tables for `ASOF JOIN` must have an ordered sequence column. This column cannot be alone in a table, and should be one of the data types: `UInt32`, `UInt64`, `Float32`, `Float64`, `Date`, and `DateTime`.
 
-You can use the following types of syntax:
+Syntax `ASOF JOIN ... ON`:
 
-- `ASOF JOIN ... ON`
+```sql
+SELECT expressions_list
+FROM table_1
+ASOF LEFT JOIN table_2
+ON equi_cond AND closest_match_cond
+```
+
+You can use any number of equality conditions and exactly one closest match condition. For example, `SELECT count() FROM A ASOF LEFT JOIN B ON A.a == B.b AND B.t <= A.t`. 
 
-    ```sql
-    SELECT expressions_list
-    FROM table_1
-    ASOF LEFT JOIN table_2
-    ON equi_cond AND closest_match_cond
-    ```
+Conditions supported for the closest match:
 
-    You can use any number of equality conditions and exactly one closest match condition. For example, `SELECT count() FROM A ASOF LEFT JOIN B ON A.a == B.b AND B.t <= A.t`. Only `table_2.some_col <= table_1.some_col` and `table_1.some_col >= table2.some_col` condition types are available. You can't apply other conditions like `>` or `!=`.
+- `table_1.some_col > table_2.some_col`
+- `table_1.some_col < table_2.some_col`
+- `table_1.some_col >= table_2.some_col`
+- `table_1.some_col <= table_2.some_col`
+- `table_2.some_col <= table_1.some_col`
 
-- `ASOF JOIN ... USING`
+Other condition types are not available.
 
-    ```sql
-    SELECT expressions_list
-    FROM table_1
-    ASOF JOIN table_2
-    USING (equi_column1, ... equi_columnN, asof_column)
-    ```
+Syntax `ASOF JOIN ... USING`:
+
+```sql
+SELECT expressions_list
+FROM table_1
+ASOF JOIN table_2
+USING (equi_column1, ... equi_columnN, asof_column)
+```
 
-    `ASOF JOIN` uses `equi_columnX` for joining on equality and `asof_column` for joining on the closest match with the `table_1.asof_column >= table2.asof_column` condition. The `asof_column` column must be the last in the `USING` clause.
+`ASOF JOIN` uses `equi_columnX` for joining on equality and `asof_column` for joining on the closest match with the `table_1.asof_column >= table_2.asof_column` condition. The `asof_column` column must be the last in the `USING` clause.
```

Comment:
> The `asof_column` column must be the last in the `USING` clause.

I think must is not the best word here. We interpret last column in use as asof column. Others - as ususal equality columns.